sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3650,6 +3650,84 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       }
     }
   }
+
+  test("SPARK-34421: Resolve temporary objects in temporary views with CTEs") {
```

Comment:
The new SQL temp view feature is not in 3.0, and I think we can remove this test, because it should pass in 3.0 without this fix.

Reply:
Yes it passes as expected. All right, removed in 92df8406026593fce14f3b6efb0aea73747ecfe5